Created test case registrationNavigation

ochaklosh opened the pull request:
Created new class Homework16
new test case function registrationNavigation() to check if after clicking on Registration Link on the sign in page, user is being redirected to registration page.

ochaklosh committed:
Created test case registrationNavigation

Created new class Homework16
new test case function registrationNavigation() to check if after clicking on Registration Link on the sign in page, user is being redirected to registration page.

Berichicko commented:
You got this problem, please correct it.

![image](https://github.com/ochaklosh/A59-QA-Automation-Framework/assets/104761482/1f787aef-2f62-4c78-8076-90c7592c6fe0)

**HINT**

![image](https://github.com/ochaklosh/A59-QA-Automation-Framework/assets/104761482/15d4c1e6-bbdc-425f-a203-e061e16f3cfe)

ochaklosh committed:
Added extends BaseTest to Homework16Test

Added extends BaseTest to Homework16Test
Updated Homework16 class to Homework16Test class
